### PR TITLE
ShopScene

### PR DESCRIPTION
**Pull request by jocke1995**

### CODE REVIEW

File: StortSpelProjekt/Engine/src/Renderer/HLSL/TransparentTexturePixel.hlsl [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+			normal.rgb,
+			baseReflectivity);
+	}
+
+	float3 ambient = float3(0.03f, 0.03f, 0.03f) * albedo;
+	finalColor += ambient;
+
+	finalColor += emissive.rgb;
+
+	float blendFactor = 0.4f;
```
Comment (fredrix96) [inline StortSpelProjekt/Engine/src/Renderer/HLSL/TransparentTexturePixel.hlsl]: unnecessary variable

**Review by fredrix96 — CHANGES_REQUESTED** [review]
The shop scene looks okay as just a simple test scene. Just delete an unnecessary variable and then it looks good

**Review by fredrix96 — DISMISSED** [review]
Looks good!

File: StortSpelProjekt/Engine/src/Renderer/HLSL/TransparentTexturePixel.hlsl (hunk 1 above)
Comment (jocke1995) [reply StortSpelProjekt/Engine/src/Renderer/HLSL/TransparentTexturePixel.hlsl]: removed

**Review by fredrix96 — APPROVED** [review]
Looks good!

**Review by OtakuFreak13 — APPROVED** [review]
Scene seems fine and game runs.

**Pull request merged**